fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/session/SessionFactory.java

```diff
@@ -133,26 +125,17 @@ public HttpSession getSession(final HttpServletRequest servletRequest) {
     protected HttpSession createSession(final HttpServletRequest servletRequest) {
 
         LOGGER.debug("Returning an authenticated session in the default workspace");
-        return  new HttpSession(repo.login(credentialsService.getCredentials(servletRequest)));
+        return  new HttpSession(txManager.create());
     }
 
     /**
      * Retrieve a JCR session from an active transaction
      *
-     * @param servletRequest the servlet request
      * @param txId the transaction id
      * @return a JCR session that is associated with the transaction
      */
-    protected HttpSession getSessionFromTransaction(final HttpServletRequest servletRequest, final String txId) {
-
-        final Principal userPrincipal = servletRequest.getUserPrincipal();
-        final String userName = userPrincipal == null ? null : userPrincipal.getName();
-
-        final FedoraSession session = batchService.getSession(txId, userName);
```

Comment:
I am wondering if we are losing anything about not capturing the userName of the agent that initiated a transaction. Is this user info used for any access control determinations, @peichman-umd ? Or used for setting anything further down, like the lastModified and created properties? Or are those populated in a different way?

Reply:
Decision from meeting: Skip for now. An User-URI can be added to the FedoraTransaction interface when we identify the need.